### DISCUSSION TITLE
Welcome to codeowners-repo Discussions!

### DISCUSSION THREAD

**Discussion by peterjuma** [pwjuma/codeowners-repo#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by peterjuma** [comment]
Vestibulum facilisis, purus nec pulvinar iaculis, ligula mi congue nunc, vitae euismod ligula urna in dolor. Quisque id odio. Sed in libero ut nibh placerat accumsan.

Etiam feugiat lorem non metus. Maecenas ullamcorper, dui et placerat feugiat, eros pede varius nisi, condimentum viverra felis nunc et lorem. Phasellus magna.
 
Praesent turpis. Fusce pharetra convallis urna. Pellentesque auctor neque nec urna.

**Comment by peterjuma** [comment]
@chaimoto Praesent nonummy mi in odio. Sed augue ipsum, egestas nec, vestibulum et, malesuada adipiscing, dui. Proin faucibus arcu quis ante.

**Comment by v-rockyofages** [comment]
Nam adipiscing. Donec elit libero, sodales nec, volutpat a, suscipit non, turpis. Quisque malesuada placerat nisl.

**Comment by v-rockyofages** [comment]
@peterjuma Nam adipiscing. Donec elit libero, sodales nec, volutpat a, suscipit non, turpis. Quisque malesuada placerat nisl.
